resources/cheatsheet.md

```diff
@@ -34,3 +34,8 @@ Delete everything in a directory with confirmation:
 ```
 rm -iR directory
 ```
+
+List information about the files:
```

Comment:
```suggestion
List the contents of the directory:
```
@smirandou confirm that this makes sense by replying or reacting  or ask a clarifying question and then I'll merge this.